Sources/Extensions/UIToolbar.swift

```diff
@@ -18,47 +18,24 @@
 
 import UIKit
 
-protocol ToolbarPickerViewDelegate: class {
-    func didTapDone(_ handler: (_ textField: UITextField) -> Void)
-}
-
-class PickerView: UIPickerView {
-    public private(set) var toolbar: UIToolbar!
-    public weak var toolBarDelegate: ToolbarPickerViewDelegate?
-    var tapDoneHandler: ((_ textField: UITextField) -> Void)!
-
-    override init(frame: CGRect) {
-        super.init(frame: frame)
-        setUpToolBar()
-    }
-
-    required init?(coder aDecoder: NSCoder) {
-        super.init(coder: aDecoder)
-        setUpToolBar()
-    }
-
-    /// Add a tool bar containing a done button on top of the picker
-    private func setUpToolBar() {
-        let toolBar = UIToolbar()
-        toolBar.barStyle = UIBarStyle.default
-        toolBar.isTranslucent = true
-        toolBar.tintColor = Theme.themeColor
-        toolBar.sizeToFit()
+extension UIToolbar {
```

Comment:
@jsaini-hw Do we really need to extend all the toolbars across the app instead of customization for our particular needs?

Reply:
@oleksii-lubianyi-epam We are not extending any existing method but giving an opportunity to extend all the toolbars if required using this method